### PR TITLE
Show stderr in red

### LINKED ISSUE #22: Show stderr in red

### PR DESCRIPTION
**Pull request by nkadonaga**
Resolve #22 

TODO
- [x] show coconut error in red

screen shot 2018-04-08 at 10 13 21 pm

- [x] show python error in red

screen shot 2018-04-08 at 10 13 45 pm

- [x] replace red highlight with red text (how to?)

### CODE REVIEW

**Comment by jcruz** [comment]
To change the font color we'll want to apply the css `color` property to the class `.ace_text-layer`. See my comment https://github.com/cs121-team-panda/coconut-interpreter-react/pull/17#issuecomment-379164161 where Mek needs to do a similar thing. The difference is that we have to make this conditional based on whether there's an error or not.

Editing the css in the inspector:
screen shot 2018-04-08 at 10 34 19 pm

**Comment by nkadonaga** [comment]
@mjenrungrot thanks for taking over! the only part that's left is replacing the red highlight with a change to red font color (just for stderr) here

**Comment by mjenrungrot** [comment]
@nkadonaga no worries!

**Comment by mjenrungrot** [comment]
image
image
image

**Comment by mjenrungrot** [comment]
@nkadonaga @jcruz Please review 🍡 
Remind that the following is the expected output
image

**Comment by mjenrungrot** [comment]
Forgot the background color
image
image

File: src/components/Output.js [hunk 1]
```diff
@@ -138,7 +165,10 @@ class Output extends Component<Props, State> {
           onLoad={this.onEditorLoad}
           readOnly
           {...aceStyleProps}
-          markers={this.state.showPython ? this.getMarkers() : []}
+          markers={
+            this.state.showPython ? this.getMarkers() : this.getTraceMarkers()
+          }
+          style={pythonErrorCall ? { color: 'red' } : {}}
```
Comment (jcruz) [inline src/components/Output.js]: This also needs to work on coconut errors so maybe change to `(pythonErrorCall || coconutErrorCall)`. This line seems a bit messy.

Comment (mjenrungrot) [reply src/components/Output.js]: I think the `coconutErrorCall` refers to just an error on the editor side.

Comment (jcruz) [reply src/components/Output.js]: But we should also color the output red
screen shot 2018-04-10 at 8 28 29 pm

**Comment by mjenrungrot** [comment]
image
image
image
image

**Review by jcruz — APPROVED** [review]

**Pull request merged**